src/polyfills/custom-elements.js

```diff
@@ -852,7 +855,68 @@ function subClass(Object, superClass, subClass) {
       value: subClass,
     },
   });
-  subClass.__proto__ = superClass;
+  setPrototypeOf(Object, subClass, superClass);
+}
+
+/**
+ * Tests whether setting '__proto__' will change the prototype chain of an
+ * object. Only needed for old IE.
+ * @return {boolean}
+ */
+function supportsUnderProto() {
+  const proto = {'test': true};
+  const obj = {};
+  obj.__proto__ = proto;
+  return !!obj['test'];
+}
+
+/**
+ * Sets the prototype chain of an object, with various fallbacks to support
+ * old IE.
+ * @param {!Object} Object
+ * @param {!Object} obj
+ * @param {!Object} prototype
+ */
+function setPrototypeOf(Object, obj, prototype) {
```

Comment:
Confusing, why pass `Object` here? Can we at least rename it so it doesn't shadow the global?